question14/main.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const wrapper = document.getElementById('js-wrapper');
+const requestBtn = document.getElementById('js-request-btn');
+const requestNumber = document.getElementById('js-number');
+const modalOpen = document.getElementById('js-modal-btn');
+const mask = document.getElementById('js-mask');
+const modal = document.getElementById('js-modal');
+const modalClose = document.getElementById('js-close');
+const inputNumber = document.getElementById("js-number");
+
+  /* MODAL OPEN */
+modalOpen.addEventListener('click',()=>{
+    modal.classList.remove('hidden');
+    mask.classList.remove('hidden');
+});
+/* MODAL close */ 
+modalClose.addEventListener('click',()=>{
+    modal.classList.add('hidden');
+    mask.classList.add('hidden');
+});
+
+mask.addEventListener('click',()=>{
+    modalClose.click();
+});
+
+async function callApi() {
+    try {
+        const res = await fetch('https://jsondata.okiba.me/v1/json/dFQLo210903231659');
+        const json = await res.json();
+        return json.data;
+    }
+    catch(error) {
+        wrapper.textContent = "データが取得できませんでした";
+        console.error(error);
+    }
+    finally{
+        modalOpen.remove();
+    }
+}
+
+async function init(){
+    const result = await callApi();
+    if (result){
+        createList(result);
+    } else {
+        wrapper.textContent = "データの表示に失敗しました";
+        console.error('データの表示に失敗しました'); 
+    }
+}
+
+requestBtn.onclick = function(){
+    if (requestNumber.value === '' || requestNumber.value === null){
```

Comment:
[IMO]
Both the empty string and NULL return false, so I thought the following writing method would be simple.
`if (! requestNumber.value === false)`

Reply:
Thank you for reviewing!  I fixed it as you said and the code is now much simpler. Thank you so much!